### DISCUSSION TITLE
Controller Mapping Shift Function

### DISCUSSION THREAD

**Discussion by RevElevenclaw** [AntiMicroX/antimicrox#898] [Ideas] [answered]
Hello, I'm new to using antimicrox. I like it a lot in it's simplicity. But I noticed there is no ability to 'shift' the controls to an alternate set up. Which is a fairly common function with other key mappers I have used. If I am in error here, and it does have the functionality, simply educate me. Otherwise consider this my official feature request.

Thanks for reading,

Rev

**Comment by BoomerangAide** [comment]
Aren't the "sets" at the bottom, and the "set change" either in Advanced within Assignments or through "Set Selector" what you're looking for?

**Reply by RevElevenclaw** [reply]
Ah, quite possibly. I will investigate further. Thank you for your response.

**Comment by MrChillings** [comment, marked as answer]
as BoomerangAide stated in the bottom you can find/make different sets (alternate set up). see image below:
![afbeelding](https://github.com/AntiMicroX/antimicrox/assets/154590517/fe80b7e7-2cd8-4734-b987-14ed324a1ce1)

as BoomerangAide also mentoined you can assign a button, trigger or pad to switch between these sets.

right click a button and navigate to set select. 
![afbeelding](https://github.com/AntiMicroX/antimicrox/assets/154590517/7453a871-827c-4abc-be68-d5beee6107f8)

or go to advanced 
![afbeelding](https://github.com/AntiMicroX/antimicrox/assets/154590517/c3d00678-f57a-4643-a44d-22f6060903d1)
To get here you left click on a button you want to assign-> dummy assign something from the keyboard lay out -> click advance

**Reply by RevElevenclaw** [reply]
Well, it was tricky to wrap my brain around at first. I needed to invert the Y mouse axis for a game without native controller support. But have non inverted for menus. So the first option shown wouldn't work. However using the second suggestion with ',' as a dummy, it works perfect with the set selector, select set 2, 2 way option. Thanks for helping me figure this out. And well done on this key mapper. Very nice indeed!

Sincerely,

Rev

**Reply by MrChillings** [reply]
when you have assigned the set selector you can remove the dummy key by pressing None. 

its just so you can enter the advance settings

**Reply by RevElevenclaw** [reply]
Oh... neat:-) Thanx for the tip.
